src/main/java/Service/DeptServiceImpl.java

```diff
@@ -0,0 +1,57 @@
+package Service;
+
+
+import org.springframework.stereotype.Service;
+import ru.homework.employee.Employee;
+
+import java.util.Comparator;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+@Service
+public class DeptServiceImpl implements DeptService {
+    private final EmployeeServiceImpl employeeServiceImpl;
```

Comment:
```suggestion
    private final EmployeeService employeeServiceImpl;
```
инъекцию надо делать по интерфейсу